src/views/blocks/parts/details.php

```diff
@@ -58,9 +59,9 @@
 <div class="tribe-events-meta-group tribe-events-meta-group-details">
 	<h3 class="tribe-events-single-section-title">
 		<?php if ( empty( $attributes['detailsTitle'] ) ) : ?>
-			<?php esc_html_e( 'Details', 'the-events-calendar' ) ?>
+			<?php esc_html_e( 'Details', 'the-events-calendar' ); ?>
 		<?php else : ?>
-			<?php echo is_array( $attributes['detailsTitle'] ) ? reset( $attributes['detailsTitle'] ) : esc_html( $attributes['detailsTitle'] ) ?>
+			<?php echo is_array( $attributes['detailsTitle'] ) ? reset( $attributes['detailsTitle'] ) : esc_html( $attributes['detailsTitle'] ); ?>
```

Comment:
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'reset'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 51e2c2ca1a15c99051be7cff0b74749d</sub>